public/app/core/selectors/navModel.ts

```diff
@@ -35,7 +37,7 @@ export const getNavModel = (navIndex: NavIndex, id: string, fallback?: NavModel,
 };
 
 function getSectionRoot(node: NavModelItem): NavModelItem {
-  return node.parentItem ? getSectionRoot(node.parentItem) : node;
+  return node.parentItem && node.parentItem !== HOME_NAV ? getSectionRoot(node.parentItem) : node;
```

Comment:
will it be overkill or useless to use deep equality comparing `node.parentItem` and `HOME_NAV`?

Reply:
hmmm, is there any time where it's not going to be the same object by reference? 🤔 otherwise it's just slower

Reply:
yeah . I'm not sure really if this is gonna be mutated somewhere :) or if we do allow node to be mutated somewhere which might cause unexpected result. Just asking as I am not sure it is mutated somewhere :)

Reply:
could switch it to look at the id instead? e.g. `node.parentItem.id !== HOME_NAV.id`? 🤔

Reply:
that should work :) thanks Ash